File: backend/models/note.js
const mongoose = require("mongoose")

const NoteSchema = new mongoose.Schema({
      date:{type:Date,default:Date.now,required:true},
      titles:{type:String,required:true },
      text:{type:String,required:true}
})

const Note = mongoose.model("Note",NoteSchema)
module.exports = Note

File: app/src/main/java/com/example/projecta/Model/Note.java
package com.example.projecta.Model;

import java.util.ArrayList;

public class Note {

    private String _id;
    private String titles ;
    private String date;
    private String text ;

    public Note( String titles,String text) {
        this.titles = titles;
        this.text = text;
    }

    public String getTitle() {
        return titles;
    }


    public String getDate() {
        return date;
    }

    public String get_id() {
        return _id;
    }

    public String getText() {
        return text;
    }




}
